pyactiveresource/activeresource.py

```diff
@@ -738,7 +738,12 @@ def _prefix(cls, options=None):
         template = Template(path)
         keys = cls._prefix_parameters()
         options = dict([(k, options.get(k, '')) for k in keys])
-        return template.safe_substitute(options)
+        prefix = template.safe_substitute(options)
+
+        prefix = re.sub(r'/+', '/', prefix)
+        if prefix.endswith('/'):
+            prefix = prefix[:-1]
```

Comment:
I think we can remove the `path = re.sub('/$', '', cls.prefix_source)` line since this does the same thing.